CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Contributing to JupyterLab
+
+If you're reading this section, you're probably interested in contributing to
+JupyterLab Metadata Service Extension. Welcome and thanks for your interest in contributing!
+
+Please take a look at the Contributor documentation, familiarize yourself with
+using JupyterLab Metadata Service Extension, and introduce yourself on the mailing list and share
+what area of the project you are interested in working on. Please see also the
+Jupyter [Community Guides](https://jupyter.readthedocs.io/en/latest/community/content-community.html).
+
+We have labeled some issues as [good first issue](https://github.com/jupyterlab/jupyterlab-metadata-service/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or [help wanted](https://github.com/jupyterlab/jupyterlab-metadata-service/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+that we believe are good examples of small, self-contained changes.
+We encourage those that are new to the code base to implement and/or ask
+questions about these issues.
+
+## General Guidelines
+
+For general documentation about contributing to Jupyter projects, see the
+[Project Jupyter Contributor Documentation](https://jupyter.readthedocs.io/en/latest/contributor/content-contributor.html) and [Code of Conduct](https://github.com/jupyter/governance/blob/master/conduct/code_of_conduct.md).
+
+All source code is written in
+[TypeScript](http://www.typescriptlang.org/Handbook). See the [Style
+Guide](https://github.com/jupyterlab/jupyterlab/wiki/TypeScript-Style-Guide).
+
+All source code is formatted using [prettier](https://prettier.io).
+When code is modified and committed, all staged files will be automatically
```

Comment:
Maybe remove this part of the paragraph down since we don't have pre commit hooks setup here.

Reply:
I am adding a precommit hook with lint-stage.